Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+﻿
+[color=#ff0000]You must be 16 or older to play. Users under 16 will be banned immediately and be unbanned when they are old enough.[/color]
+[color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large. They will of course be held fully accountable for their actions if they exercise this privilege.
+
+[color=#ff0000]Some documents available on the wiki at wiki.spacestation14.io such as Space Law, the Standard Operating Procedure, and the Alert Procedure will be mentioned here and are expected to be followed.[/color]
+
+Core Rules
+
+[color=#a4885c]0.[/color] Take these rules as they are to be intended and not as they are written literally. Rule skirting and lawyering is also bannable at the admin teams discretion.
+[color=#a4885c]1.[/color] Don’t be a dick. We’re all here to have fun. Antagonists have leeway here since ruining the round is kind of their job, but try to keep it interesting if you’re antag.
+    - Be considerate of what other players want. If everyone is dead, don’t stall the end of the round to finish your personal projects (such as building a shuttle).
+[color=#a4885c]2.[/color] These are English servers. Do not speak other languages in IC or OOC, our staff cannot moderate it and you will be asked to stop.
+[color=#a4885c]3.[/color] These servers are “Roleplay Expected”, meaning you should remain in character at all times unless speaking in OOC chat or local OOC (LOOC) chat. Basic roleplaying standards like not using emoji/emoticons or textspeak (:), :D, xD, lol, etc.) in speech, and using real names for characters, are expected to be followed.
+[color=#a4885c]4.[/color] Follow basic common sense–you cannot murder someone because they slipped you on a banana.
+    - This isn’t a deathmatch, be humane.
+    - What is considered a reasonable reason to escalate conflict is listed below under ESCALATION RULES and is at the discretion of admins.
```

Comment:
Might want to specially format `ESCALATION RULES` ?

This is also probably a good area to mention that a mild amount of overescalation is an IC issue.